### PR TITLE
Added support for a+ and w+ modes (files)

### PR DESCRIPTION
**Pull request by yashpatel5400**
Supports missing modes of writing to files that are available in the Python standard libraries but not current supported here

### CODE REVIEW

File: runtime/file.go [hunk 1]
```diff
@@ -127,6 +127,11 @@ func fileInit(f *Frame, o *Object, args Args, _ KWArgs) (*Object, *BaseException
 		flag = os.O_RDONLY
 	case "r+", "r+b":
 		flag = os.O_RDWR
+	// difference between r+ and a+ is that a+ automatically creates file
```
Comment (trotterdylan) [inline runtime/file.go]: Please use full sentences with capitalization and period for comments

**Comment by trotterdylan** [comment]
This is great! Thanks for the PR. One minor coherent.

File: runtime/file.go (hunk 1 above)
Comment (yashpatel5400) [inline runtime/file.go]: Thanks! Just pushed it with the correct formatting

**Comment by corona10** [comment]
@yashpatel5400 FYI, `gofmt -w runtime/*.go` will help you for formatting problem 👍

**Pull request merged**